services/dependabot/dependabot.tester.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const Joi = require('joi');
+const ServiceTester = require('../service-tester');
+const { isIntegerPercentage } = require('../test-validators');
+const colorscheme = require('../../lib/colorscheme.json');
+const mapValues = require('lodash.mapvalues');
+const colorsB = mapValues(colorscheme, 'colorB');
+
+const t = new ServiceTester({ id: 'dependabot', title: 'Dependabot' });
+module.exports = t;
+
+t.create('semver compatibility (valid)')
+  .get('/semver/bundler/puma.json')
```

Comment:
You have to add `?style=_shields_test` at the end of url (`/semver/bundler/puma.json?style=_shields_test`) if you want to test colors. Without it JSON response does not contain `colorB` property. `Joi.equal(colorsB.brightgreen, colorsB.orange)` is true when property is missing. `Joi.equal(colorsB.brightgreen, colorsB.orange).required()` means that property is required.